packages/frontend-shared/src/locales/en-US.json

```diff
@@ -598,7 +598,7 @@
       "description": "Complete the steps below to migrate your project to Cypress 10",
       "step1": {
         "title": "Rename existing specs",
```

Comment:
In the screenshot, this title is changed, but it's not here?

Reply:
Oh, let's use "Migrate existing specs" for the title. It's the perfect description for what we are doing, it's the migration UI, after all.